README.md

```diff
@@ -163,6 +163,13 @@ nix-shell> nixpkgs-review comments
 `nixpkgs-review` will by default use [nix-output-monitor](https://github.com/maralorn/nix-output-monitor) if found in `$PATH`.
 If you have `nom` installed but don't want to use it, you can pass `nix` to `--build-graph` to use `nix build` instead of `nom build`.
 
+Some pull requests may require configuration for nixpkgs to test out. You can
+use the `--extra-nixpkgs-config` flag to supply extra configuration for nixpkgs.
+
+```console
+$ nixpkgs-review pr 37242 --extra-nixpkgs-config '{ cudaSupport = true; }'
```

Comment:
Since we are now adding more option that influence how nix evaluation is done, can we add this to `report.md` as well? This would help the community a lot if some posts a build result for reproduction.

Reply:
It does go into the report:
![image](https://user-images.githubusercontent.com/9720532/222680812-5dacacfb-2861-4d28-b323-526b96f31648.png)